ansible/roles/sunbird-auth-deploy/files/python-keycloak-0.12.0/clients.json

```diff
@@ -0,0 +1,266 @@
+{
+    "clients" : [
+        {
+            "clientId": "google-auth-android",
+            "rootUrl": "https://dev.sunbirded.org",
```

Comment:
can the DNS be variablized and not hardcoded.

Reply:
Fixed